src/Agent/MyStatistics.vue

```diff
@@ -15,29 +15,29 @@
         <b-table style="margin-top:10px" small bordered
           :items="statistics"
           :fields="fields">
-          <template slot="HEAD_ciq" scope="data">
-            <div style="text-align:center; font-weight: normal; color:#969696">{{data.label}}</div>
+          <template slot="HEAD_ciq" slot-scope="data">
+            <div class="table-header-text-center">{{data.label}}</div>
```

Comment:
Better to use b-table thClass, don't you think? Also, why do you need slots for headers at all?

Reply:
because it was one possible option